src/toTransform.js

```diff
@@ -62,8 +62,20 @@
     }
   }
 
+  function checkPathLoop (path) {
+    var pathInput = path.input.replace(/[,\s]+$/g, '');
+
+    var lastPathInput = pathInput[pathInput.length - 1];
+    if (lastPathInput === 'z' || lastPathInput === 'Z') {
+      return true;
+    }
+
+    return false;
```

Comment:
Just return the if condition. There should be a toLower() or similarly named method as well.